server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.ts

```diff
@@ -1403,7 +1403,7 @@ export class AgenticChatController implements ChatHandlers {
         }
 
         // These are errors we want to show custom messages in chat for.
-        if (err.code === 'QModelResponse' || err.code === 'MaxAgentLoopIterations') {
+        if (err.code === 'QModelResponse' || err.code === 'MaxAgentLoopIterations' || err.code == 'InputTooLong') {
```

Comment:
At some point should we just refactor this out to be something like:
```
['QModelResponse', 'MaxAgentLoopIterations', 'InputTooLong'].includes(err.code)
```
or even pull them out into a function or something

Reply:
Yeah we should